src/components/contextual/pages/pool/PoolChart.vue

```diff
@@ -381,6 +382,11 @@ const defaultChartData = computed(() => {
   return { title, value: chartData.value.defaultHeaderStateValue };
 });
 
+const showFxPoolWarning = computed(() => {
```

Comment:
Since this warning is Xave specific we should probably limit this conditional to only Xave pools too.

Reply:
No, I don't think so. I think the pool factories are permissionless so anyone could technically create an FX pool. Is that right @markusbkoch?

Reply:
Talked to Markus and I think it's fine to leave this conditional as it is.